BuyBitcoin/templates/setup_bank.html

```diff
@@ -5,7 +5,8 @@
 <script src="https://cdn.plaid.com/link/v2/stable/link-initialize.js"></script>
 
 <button id="link-button">Open Link</button>
-
+<form>{% csrf_token %}
```

Comment:
what's this for

Reply:
CSRF token was missing. This makes sure that it is present for the plaid token sending

Reply:
Doesn't line 25 have the csrf token?

Reply:
No, we need to make sure that the CSRF token is provided as a cookie. Line 25 only grabs it once it is present.